backend_qemu.go

```diff
@@ -224,8 +224,11 @@ func (b qemuBackend) StartQemu(kvm bool) (bool, error) {
 	qemuargs = append(qemuargs, "-machine", qemuMachine.machine)
 	console := fmt.Sprintf("console=%s", qemuMachine.console)
 	kernelargs := []string{console, "panic=-1",
+		"nosplash",
 		"systemd.unit=fakemachine.service"}
 
+	kernelargs = append(kernelargs, "nosplash")
+
```

Comment:
I'm seeing double? Seems like we're adding the argument twice.

Should this be dependant on the "show-boot" option? In the default use-case I don't want to see any kernel/boot log, which plymouth hides nicely.